generators/python-v2/ast/src/ClassInstantiation.ts

```diff
@@ -1,15 +1,29 @@
 import { BaseInvocation } from "./BaseInvocation";
 import { Reference } from "./Reference";
+import { Writer } from "./core/Writer";
 
 export declare namespace ClassInstantiation {
     interface Args extends Omit<BaseInvocation.Args, "reference"> {
         /* A reference to the class that you'd like to instantiate */
         classReference: Reference;
+        /* The name of the variable to assign the instantiated class to */
+        assignTo?: string;
     }
 }
 
 export class ClassInstantiation extends BaseInvocation {
-    constructor({ classReference, ...args }: ClassInstantiation.Args) {
+    private assignTo: string | undefined;
+
+    constructor({ classReference, assignTo, ...args }: ClassInstantiation.Args) {
         super({ reference: classReference, ...args });
+        this.assignTo = assignTo;
+    }
+
+    public write(writer: Writer): void {
+        if (this.assignTo) {
+            writer.write(this.assignTo);
+            writer.write(" = ");
+        }
+        super.write(writer);
```

Comment:
Rather than having the `ClassInstantiation` own the assignee, should we think about designing this in a way such that we have an `Assign` node that has knowledge of the left and right operands?

Reply:
Absolutely, much better that way - I'll follow up with that change. I'll probably call it Assignment.

Reply:
>  I'll probably call it Assignment.

Sure, I'm not too opinionated on this part. The standard library calls it [ast.Assign](https://docs.python.org/3/library/ast.html#ast.Assign).

Reply:
Ah nice, let's go with `Assign` - I like maintaining consistency with the standard lib naming conventions.